tools/openocd/defs.bzl

```diff
@@ -92,20 +94,66 @@ def _openocd_debug_server_impl(ctx):
     if ctx.attr.transport:
         transport_string = "-c \"transport select %s\"" % (ctx.attr.transport)
     script_template = """
-{openocd} {interface_config_string} {transport_string} -c "adapter_khz {programmer_frequency}" {chip_config_string}
+{openocd} {interface_config_string} {transport_string} -c "adapter speed {programmer_frequency}" {chip_config_string}
 
 """
 
-    script = ctx.actions.declare_file("%s.sh" % ctx.label.name)
-    script_content = script_template.format(
-        openocd = ctx.file._openocd.short_path,
-        interface_config_string = interface_config_string,
-        chip_config_string = chip_config_string,
-        programmer_frequency = ctx.attr.programmer_frequency,
-        transport_string = transport_string,
-    )
+    script_content = ""
+    if is_windows:
+        # A bit of a hacky way to do this... The issue is Windows does not support
```

Comment:
Part of the reason why this isn't properly supported on windows is because I gave up on dealing with symlinks. My primary dev environment is linux based so this became a low priority.